articles/batch/batch-upgrade-policy.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Provision a pool with Auto OS Upgrade
+description: Learn how to create a Batch pool with Auto OS Upgrade so that customers can have control over their OS upgrade strategy to ensure safe, workload-aware OS upgrade deployments.
+ms.topic: how-to
+ms.date: 10/30/2023
+ms.custom: 
+---
+
+# Create an Azure Batch pool with Auto OS Upgrade
+
+> [!IMPORTANT]
+> - Support for pools with Auto OS Upgrade in Azure Batch is currently in public preview, and is currently controlled by an account-level feature flag. If you want to use this feature, please start a [support request](../azure-portal/supportability/how-to-create-azure-support-request.md) and provide your batch account to request its activation.
+> - This preview version is provided without a service level agreement, and it's not recommended for production workloads. Certain features might not be supported or might have constrained capabilities.
+> - For more information, see [Supplemental Terms of Use for Microsoft Azure Previews](https://azure.microsoft.com/support/legal/preview-supplemental-terms/).
+
+When you create an Azure Batch pool, you can provision the pool with nodes that have Auto OS Upgrade enabled. This article explains how to set up a Batch pool with Auto OS Upgrade.
+
+## Why use Auto OS Upgrade?
+
+Auto OS Upgrade is used to implement an automatic operating system upgrade strategy and control within Azure Batch Pools. Here are some reasons for using Auto OS Upgrade:
+
+- **Security.** Auto OS Upgrade ensures timely patching of vulnerabilities and security issues within the operating system image, thereby enhancing the security of compute resources. This helps prevent potential security vulnerabilities from posing a threat to applications and data.
+- **Minimized Availability Disruption.** Auto OS Upgrade is used to minimize the availability disruption of compute nodes during OS upgrades. This is achieved through task-scheduling-aware upgrade deferral and support for rolling upgrades, ensuring that workloads experience minimal disruption.
+- **Flexibility.** Auto OS Upgrade allows you to configure your automatic operating system upgrade strategy, including percentage-based upgrade coordination and rollback support. This means you can customize your upgrade strategy to meet your specific performance and availability requirements.
+- **Control.** Auto OS Upgrade provides you with control over your operating system upgrade strategy to ensure secure, workload-aware upgrade deployments. You can tailor your policy configurations to meet the specific needs of your organization.
+
+In summary, the use of Auto OS Upgrade helps you achieve automatic operating system upgrades, leading to improved security, minimized availability disruptions, and greater control and flexibility for your workloads. This is crucial for maintaining the health and security of compute resources.
```

Comment:
"In summary, the use of Auto OS Upgrade helps improve security, minimize availability disruptions, and provide both greater control and flexibility for your workloads."